packages/desktop-client/src/components/mobile/accounts/AccountTransactions.tsx

```diff
@@ -140,12 +174,36 @@ function AccountName({ account, pending, failed }) {
   );
 }
 
-function TransactionListWithPreviews({ account }) {
-  const [currentQuery, setCurrentQuery] = useState();
+function NameOnlyHeader({ accountName }: { readonly accountName: string }) {
+  return (
+    <View
+      style={{
+        flexDirection: 'row',
+      }}
+    >
+      <Text style={{ ...(styles.lineClamp(2) as CSSProperties) }}>
+        {accountName}
+      </Text>
+    </View>
+  );
+}
+
+function TransactionListWithPreviews({
+  account,
+  accountId,
+  accountName,
+}: {
+  readonly account?: AccountEntity;
+  readonly accountId?: string;
+  readonly accountName: string;
+}) {
+  const [currentQuery, setCurrentQuery] = useState<Query>();
   const [isSearching, setIsSearching] = useState(false);
   const [isLoading, setIsLoading] = useState(true);
-  const [transactions, setTransactions] = useState([]);
-  const prependTransactions = usePreviewTransactions();
+  const [transactions, setTransactions] = useState<
+    ReadonlyArray<TransactionEntity>
+  >([]);
+  const prependTransactions = usePreviewTransactions(() => {});
```

Comment:
I think `usePreviewTransacton` arg should be made optional so that we don't need to pass in an empty lambda.